hazelcast/src/main/java/com/hazelcast/core/DistributedObjectEvent.java

```diff
@@ -90,6 +94,10 @@ public DistributedObject getDistributedObject() {
         return distributedObject;
     }
 
+    public UUID getSource() {
```

Comment:
javaDoc?

Reply:
i found calling this method as `getSourceUuid` more explicit.

Reply:
I check the other events used in the system. e.g. `EntryEvent` uses the word `source`. I only found that the `Invalidation` using the `sourceUuid`, hence I am not sure if we should rename this. I will get a second opinion.

Reply:
To me, DistributedObjectEvent should extend EventObject as our other public events do. And in that case, we have to implement `getSource`

Reply:
not a big issue, just noted that in favor of explicitness.

Reply:
OK, I made DistributedObjectEvent extend EventObject.